vsphere/internal/helper/guestoscustomizations/customizations_helper.go

```diff
@@ -601,6 +613,7 @@ func expandCustomizationIdentification(d *schema.ResourceData, prefix string) ty
 		JoinWorkgroup: d.Get(prefix + "workgroup").(string),
 		JoinDomain:    d.Get(prefix + "join_domain").(string),
 		DomainAdmin:   d.Get(prefix + "domain_admin_user").(string),
+		DomainOU:      d.Get(prefix + "domain_ou").(string),
```

Comment:
```suggestion
        	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 8, Minor: 0, Patch: 2}) {
		DomainOU:      d.Get(prefix + "domain_ou").(string),
	}
```

Reply:
@tenthirtyam are you sure? there is no `viapi` object in the `expandCustomizationIdentification` function

Reply:
i've implemented that diffidently

```go
	obj := types.CustomizationIdentification{
		JoinWorkgroup: d.Get(prefix + "workgroup").(string),
		JoinDomain:    d.Get(prefix + "join_domain").(string),
		DomainAdmin:   d.Get(prefix + "domain_admin_user").(string),
	}
	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 8, Minor: 0, Patch: 2}) {
		obj.DomainOU = d.Get(prefix + "domain_ou").(string)
	}
```